### ISSUE TITLE
Static Qt build: compiler.runtime is not allowed with gcc

### ISSUE THREAD

**Issue by jgeudens** [bincrafters/community#570]
### Description of Problem, Request, or Question
I receive an error when attempting to do a static Qt build on Windows 10 with GCC (MinGW).

> ERROR: Qt/5.9.7@bincrafters/stable: Error in build() method, line 246
>         if self.settings.compiler.runtime == "MT" or self.settings.compiler.runtime == "MTd":
>         ConanException: 'settings.compiler.runtime' doesn't exist for 'gcc'
> 'settings.compiler' possible configurations are ['exception', 'libcxx', 'threads', 'version']

The issue appears to be that a compiler/runtime setting is used while this is not allowed by GCC.

### Package Details (Include if Applicable)
* Package Name/Version: **conan-qt/stable/5.9.7**
* Operating System: **Windows 10**
* Compiler+version: **gcc-5.3**

### Steps to reproduce (Include if Applicable)

My conan command line is:
`conan install .. -g qmake -g virtualenv -s os=Windows -s compiler=gcc -s compiler.version=5.3 -s compiler.libcxx=libstdc++11`

Conan file
```
[requires]
Qt/5.9.7@bincrafters/stable

[generators]
qmake

[options]
Qt:shared=False
Qt:qtserialbus=True
```

**Comment by jgeudens** [comment]
Thank you for the quick response and fix.

Is the testing/5.9.7 package available through bintray?

**Comment by ericLemanissier** [comment]
Unfortunately testing recipe are never uploaded to bintray. You can clone https://github.com/bincrafters/conan-qt/compare/testing/5.9.7 and then create the package from your local copy.

**Comment by ericLemanissier** [comment]
The fix is available now on stable/5.9.7 on bintray, so I'm closing this issue, I'll push the change on the other branches tomorrow.
